data_loader.py

```diff
@@ -264,12 +265,18 @@ def load(self, file_list, cheat_mode, dry_run, loading_mode, wipe_db, max_violat
             if not neo4j_uri:
                 self.log.error('No Neo4j URI specified for backup, abort loading!')
                 sys.exit(1)
-            backup_name = datetime.datetime.today().strftime(DATETIME_FORMAT)
             host = get_host(neo4j_uri)
-            restore_cmd = backup_neo4j(backup_folder, backup_name, host, self.log)
-            if not restore_cmd:
-                self.log.error('Backup Neo4j failed, abort loading!')
-                sys.exit(1)
+            if self.database_type == "neo4j":
+                backup_name = datetime.datetime.today().strftime(DATETIME_FORMAT)
+                restore_cmd = backup_neo4j(backup_folder, backup_name, host, self.log)
+                if not restore_cmd:
+                    self.log.error('Backup Neo4j failed, abort loading!')
+                    sys.exit(1)
+            elif self.database_type == "memgraph":
+                backup_name = backup_memgraph(backup_folder, self.memgraph_snapshot_dir, self.log)
+                if not backup_name:
+                    self.log.error('Backup Memgraph failed, abort loading!')
+                    sys.exit(1)
```

Comment:
Please provide an error message if database_type is neither.

Reply:
The error message is added